test/test_sag.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from fireapi.model import Sag
+
+
+@pytest.mark.skip(reason="Sag cannot be inseted atm because of missing mapping of sagstype")
```

Comment:
spelling error (inseted)

Reply:
Is that to draw attention away from the fact that `sqlalchemy` is still missing?

Reply:
:pineapple: